src/documentation/pages/components/tooltip/examples/_base_tooltip.vue

```diff
@@ -0,0 +1,66 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base-tooltip" padding="-p--0" :tabs="exampleTabs">
+    .-p--3(slot="example")
```

Comment:
paddings issue here as well, please use the default -p--3 padding in all corresponding examples. I.e:

```
<ComponentExample title="Base" id="base-tooltip" :tabs="exampleTabs">
    button(slot="example" data-tooltip='Your tooltip text on a button').chi-button Hover me to see tooltip
```